tokio-stream/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.1.16 (September 5th, 2024)
+
+- stream: add `next_many` and `poll_next_many` to `StreamMap` (#6409)
+- stream: make stream adapters public (#6658)
+- readme: add readme for tokio-stream (#6456)
+- chore: increase MSRV to 1.70 (#6645)
```

Comment:
Can you move this comment to the top, matching the changelog for 0.1.15?

Reply:
Done.